### DISCUSSION TITLE
Custom Webhooks

### DISCUSSION THREAD

**Discussion by mlabouardy** [tailwarden/komiser#839] [Announcements]
## What's Changed
* Fix AWS sao paulo region lat/long by @mlabouardy in https://github.com/tailwarden/komiser/pull/828
* Fix AWS EC2 costs when they're stopped by @naisayer in https://github.com/tailwarden/komiser/pull/830
* Fix number format in cost explorer by @mlabouardy in https://github.com/tailwarden/komiser/pull/831
* chore(deps): bump github.com/gin-gonic/gin from 1.9.0 to 1.9.1 by @dependabot in https://github.com/tailwarden/komiser/pull/832
* enhancement: add new ascii art logo by @dipankardas011 in https://github.com/tailwarden/komiser/pull/833
* fix: Dockerfile MAINTAINER label by @dipankardas011 in https://github.com/tailwarden/komiser/pull/835
* Custom Webhook Integration by @ShubhamPalriwala in https://github.com/tailwarden/komiser/pull/826

## New Contributors
* @dipankardas011 made their first contribution in https://github.com/tailwarden/komiser/pull/833

**Full Changelog**: https://github.com/tailwarden/komiser/compare/v3.0.17...v3.0.18

<hr /><em>This discussion was created from the release <a href='https://github.com/tailwarden/komiser/releases/tag/v3.0.18'>Custom Webhooks</a>.</em>

**Comment by dipankardas011** [comment]
Hi Everyone 😄
